src/sevcore_linux.cpp

```diff
@@ -771,17 +771,19 @@ int SEVDevice::generate_vcek_ask(const std::string output_folder,
     return cmd_ret;
 }
 
-int SEVDevice::request_platform_status(snp_platform_status_buffer &plat_status) {
-    int ioctl_return = 0;
-    snp_platform_status_cmd_buf buf = { .status_p_addr = (uint64_t)&plat_status };
-    sev_ioctl(SEV_SNP_PLATFORM_STATUS, &buf, &ioctl_return);
+int SEVDevice::request_platform_status(snp_platform_status_buffer *plat_status) {
```

Comment:
Isn't it preferred to use references instead of raw pointers?

Reply:
looked at
```
int SEVDevice::get_platform_es(void *data)
int SEVDevice::platform_status(uint8_t *data)
```

Reply:
Yes, these are a few cases we are using pointers, though we try to avoid using them where possible. Where this is being forwarded to a C-IOCTL, it might make sense, but the linux spec expects the value to be cast to a `uint64_t`, anyway. I am open to discussing it.